firmware/user/modes/mode_tunernome.c

```diff
@@ -316,6 +321,11 @@ void ICACHE_FLASH_ATTR tunernomeEnterMode(void)
     enableDebounce(true);
 
     InitColorChord();
+
+    tunernome->exitTimeAccumulatedUs = 0;
```

Comment:
you don't have to set things in the tunernome struct to 0, since the struct is calloc'd

Reply:
Functionally you're spot on. It's good to explicitly see though, like code-as-a-comment.